Installation of bbufr fails

Tried to make a test built of the VM today. With the current code, the built fails when calling `make` and `make install` from `install_baltrad_bbufr.sh` with what seems an issue with `aclocal-1.9`:

```
==> default: cd . && /bin/bash /home/vagrant/tmp/bbufr/missing --run aclocal-1.9
==> default: /home/vagrant/tmp/bbufr/missing: line 81: aclocal-1.9: command not found
==> default: WARNING: 'aclocal-1.9' is missing on your system.
==> default:          You should only need it if you modified 'acinclude.m4' or
==> default:          'configure.ac' or m4 files included by 'configure.ac'.
==> default:          The 'aclocal' program is part of the GNU Automake package:
==> default:          <http://www.gnu.org/software/automake>
==> default:          It also requires GNU Autoconf, GNU m4 and Perl in order to run:
==> default:          <http://www.gnu.org/software/autoconf>
==> default:          <http://www.gnu.org/software/m4/>
==> default:          <http://www.perl.org/>
==> default: Makefile:176: recipe for target 'aclocal.m4' failed
==> default: make: *** [aclocal.m4] Error 127
==> default: + make install
==> default: cd . && /bin/bash /home/vagrant/tmp/bbufr/missing --run aclocal-1.9
==> default: /home/vagrant/tmp/bbufr/missing: line 81: aclocal-1.9: command not found
==> default: WARNING: 'aclocal-1.9' is missing on your system.
==> default:          You should only need it if you modified 'acinclude.m4' or
==> default:          'configure.ac' or m4 files included by 'configure.ac'.
==> default:          The 'aclocal' program is part of the GNU Automake package:
==> default:          <http://www.gnu.org/software/automake>
==> default:          It also requires GNU Autoconf, GNU m4 and Perl in order to run:
==> default:          <http://www.gnu.org/software/autoconf>
==> default:          <http://www.gnu.org/software/m4/>
==> default:          <http://www.perl.org/>
==> default: Makefile:176: recipe for target 'aclocal.m4' failed
==> default: make: *** [aclocal.m4] Error 127
The SSH command responded with a non-zero exit status. Vagrant assumes that this means the command failed. The output for this command should be in the log above. Please read the output to determine what went wrong.
```

--- a/provision_scripts/install_baltrad_bbufr.sh
+++ b/provision_scripts/install_baltrad_bbufr.sh
@@ -7,6 +7,7 @@ set -x
 #sudo apt-get install -qq libproj0
 sudo apt-get install -qq proj-bin
 sudo apt-get install -qq libproj-dev
+sudo apt-get install -qq autoconf
 
 # install bbufr from source
 cd ~
@@ -16,6 +17,7 @@ fi
 cd tmp
 git clone --depth=1 git://git.baltrad.eu/bbufr.git
 cd bbufr/
+autoreconf -f -i
 ./configure --prefix=/opt/baltrad/bbufr LDFLAGS="-L/usr/lib/x86_64-linux-gnu/hdf5/serial" CFLAGS="-I/usr/include/hdf5/serial"
 make
 make install

